CIP-1694/README.md

```diff
@@ -0,0 +1,664 @@
+
+#### New Stake Distribution for DReps
+
+In addition to the existing per-stake-credential distribution and the per-stake-pool distribution, we will now have a new per-DRep stake distribution.
+
+This distribution will determine how much stake is backed by each `yes` vote from a DRep. The exact stake snapshot used for block production by the SPOs will also be used for the DReps for voting.
+
+#### Definitions surrounding Voting Stake
+
+We define some notions around voting stake:
+
+* Lovelace contained in a transaction output is considered **active for voting** if:
+  * It contains a registered stake credential.
+  * The stake credential has delegated its voting rights to a registered DRep.
+* Relative to some given percentage `P`, we say that there is **sufficient active voting stake** if
+  the ratio of the *total active voting stake* to the *total stake in circulation* is at least `P`.
+* Relative to a percentage `P`, a DRep (SPO) **vote threshold has been met** if the sum of the relative stake controlled by the DReps (SPOs) voting 'yes' to a governance action minus the sum of the relative stake controlled by the DReps (SPOs) voting 'no' is at least `P`.
+
+> **Note**
+> There are several alternative definitions for "the total stake in circulation":
+>  1. The sum of the UTxO and the rewards accounts.
+>  2. The sum of the UTxO, the rewards accounts, the fee pot, and the deposit pot.
+>  3. The total ADA supply (i.e. 45 billion ADA) minus the reserves[^3].
+>  We leave the choice open for discussion.
+
+#### New Protocol Parameters
+
+New protocol parameters will be needed for the following:
+
+* the governance action deposit amount
+* governance action expiration (as a number of epochs from the current epoch)
+* thresholds for treasury withdrawals
+* each of the ratification thresholds
+
+As described above, some  of these will be updatable; others will be hard coded.
+
+// TODO: Decide on the initial parameter values and whether they should be updatable.
+
+// TODO: Decide on coherence conditions on the voting thresholds. For example, the threshold for a motion of no-confidence should arguably be higher than that of a minor treasury withdrawal.
+
+In addition, the initial Constitutional Committee and the initial Constitution will need to be defined as part of a bootstrap process.
+
+#### Changes to the existing ledger rules
+
+* The `PPUP` transition rule will be rewritten and moved out of the `UTxO` rule and into the `LEDGER` rule as a new `TALLY` rule.
+
+  It will process the governance actions and the votes, ratify them, and stage governance actions for enactment in the current or next epoch, as appropriate.
+
+* The `NEWEPOCH` transition rule will be modified.
+* The `MIR` sub-rule will be removed.
+* A new `ENACTMENT` rule will be called immediately after the `EPOCH` rule. This rule will enact governance actions that have previously been ratified.
```

Comment:
Elsewhere in the document, the rule is called `RATIFY`.

Reply:
great catch, thank you, I'll get this once I've collected a few other edits.

Reply:
>Having some kind of educational or other requirement for someone to become a DRep has many of the same problems that such a requirement has for lawmakers in government.

And those problems would be? Enlighten me. Tired of people in this community saying x won't work and then not explaining why. Could at least drop a link.